net/iconik/distinfo

```diff
@@ -0,0 +1,3 @@
+TIMESTAMP = 1567714449
+SHA256 (iconik_storage_gateway-latest.txz) = 6b241a5d39d794a2b94e9743728614349798d0d65efa21a237c89395c0c40c79
```

Comment:
Lets try to find out if they can provide a link for a static file, that will not change over time (e.g. no latest)

Reply:
I'll email Joe to see if we can make that happen